### PR TITLE
Update using-the-assets-builder.md

### PR DESCRIPTION
**Pull request by carolinamenezes**
Improve step-by-step and edit image

### CODE REVIEW

File: docs/en/Recipes/development/using-the-assets-builder.md [hunk 1]
```diff
@@ -9,43 +9,50 @@ git: "https://github.com/vtex-apps/io-documentation/blob/master/docs/en/Recipes/
...
+4. Now you're able to use your assets within store theme blocks that uses media, such as an [Image](https://vtex.io/docs/components/all/vtex.store-components@3.115.3/image/) block, an [Info Card](https://vtex.io/docs/components/all/vtex.store-components@3.116.0/infocard/), etc. To do that, you must use the desired asset path as the value of a given prop of your block. For example, for an Image block, you'll use `assets/{your-image}.{jpg/png/gif}` as the `src` value:
 
+```JSON
+"image": {  
+    "props": {  
+      "src": "assets/myimage.png"  
+    }  
+}
+```
+Our builder will automatically look for the path `assets/{your-image}.{jpg/png/gif}` of your asset in the **File Manager** to replace its value with its URL.
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/using-the-assets-builder.md:53]:
```suggestion

Once the asset path was added to the prop and you save the changes performed, Assets Builder will automatically work to save it in the VTEX IO database and then generate a URL for it, which will be considered by the platform during the theme rendering.
```

File: docs/en/Recipes/development/using-the-assets-builder.md [hunk 2]
```diff
@@ -9,46 +9,53 @@ git: "https://github.com/vtex-apps/io-documentation/blob/master/docs/en/Recipes/
...
+      "src": "assets/myimage.png"  
+    }  
+}
+```
 
+Once the asset path was added to the prop and you save the changes performed, Assets Builder will automatically work to save it in the VTEX IO database and then generate a URL for it, which will be considered by the platform during the theme rendering.
 <div class="alert alert-warning">
-Remember to include the folder hierarchy in the asset path, such as: <code>assets/events/vtex-day.png</code>.
+If you created subfolders inside the `assets` folder, remember to include the folder hierarchy in the asset path, such as:  `assets/events/vtex-day.jpg`.
 </div>
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/using-the-assets-builder.md:56]: <div class="alert alert-warning">
Remember to include the folder hierarchy in the asset path, such as: <code>assets/events/vtex-day.png</code>.
If you created subfolders inside the `assets` folder, remember to include the folder hierarchy in the asset path, such as:  `assets/events/vtex-day.jpg`.
</div>
Once the asset path was added to the prop and you save the changes performed, Assets Builder will automatically work to save it in the VTEX IO database and then generate a URL for it, which will be considered by the platform during the theme rendering.

File: docs/en/Recipes/development/using-the-assets-builder.md [hunk 3]
```diff
@@ -9,46 +9,53 @@ git: "https://github.com/vtex-apps/io-documentation/blob/master/docs/en/Recipes/
...
 
-2. Create an `assets` folder in your app *root* directory, to insert and manage the desired assets:
+2.  In the _root_  directory of your app, create an  `assets`  folder to store and manage your store's assets, such as images:
 
-![assets-folder](https://user-images.githubusercontent.com/52087100/75712413-a6252480-5ca6-11ea-8ef6-7c54752d451f.png)
 
-3. In the `assets` folder, add the desired asset files as you want as showed above. You can create subfolders within the `assets` folder and better organize the assets used by the theme blocks. 
+3. Then, add the desired asset files in the  `assets` folder. Notice that you can create subfolders within the  `assets`  folder to better organize the assets used by the theme blocks, as shown below:
+
+![assets-folder](https://user-images.githubusercontent.com/60782333/83685560-3e40eb80-a5bf-11ea-9ea1-d443bce21b11.png)
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/using-the-assets-builder.md:39]:
```suggestion
![assets-folder](https://user-images.githubusercontent.com/60782333/83685560-3e40eb80-a5bf-11ea-9ea1-d443bce21b11.png)

<div class="alert alert-warning">
If you created subfolders inside the Assets folder, remember to include the folder hierarchy in the asset path, such as:  <code>assets/events/vtex-day.jpg</code>.
</div>
```

File: docs/en/Recipes/development/using-the-assets-builder.md [hunk 4]
```diff
@@ -9,46 +9,54 @@ git: "https://github.com/vtex-apps/io-documentation/blob/master/docs/en/Recipes/
...
+
+```JSON
+"image": {  
+    "props": {  
+      "src": "assets/myimage.png"  
+    }  
+}
+```
+
+Once the asset path was added to the prop and you save the changes performed, Assets Builder will automatically work to save it in the VTEX IO database and then generate a URL for it, which will be considered by the platform during the theme rendering.
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/using-the-assets-builder.md:57]:
```suggestion
Once the asset path was added to the prop and you save the changes performed, Assets Builder will automatically work to save it in the VTEX IO File Manager and then generate a URL for it, which will be considered by the platform during the theme rendering.
```

**Pull request merged**